### PR TITLE
bug(TDI-38517): Make i18n for tSnowflake components

### PR DESCRIPTION
**Pull request by dmytro-sylaiev**
* Create .properties file for each class which need i18n

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [x] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeConstants.java [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+
+    private static final I18nMessages i18nMessages = GlobalI18N.getI18nMessageProvider()
+            .getI18nMessages(SnowflakeConstants.class);
+
     public static final String TALEND_DEFAULT_DATE_PATTERN = "yyyy-MM-dd";
 
     public static final String TALEND_DAFEULT_TIMESTAMP_PATTERN = "yyyy-MM-dd'T'HH:mm:ss";
 
-    public static final String INCORRECRT_SNOWFLAKE_ACCOUNT_MESSAGE = " Incorrect Snowflake Account was specified..";
+    public static final String INCORRECRT_SNOWFLAKE_ACCOUNT_MESSAGE = i18nMessages.getMessage("error.incorrectAccount");
```
Comment (francisu) [inline components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeConstants.java]: Can you fix the typo in this constant as well?

File: components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSink.java [hunk 2]
```diff
@@ -38,8 +42,8 @@ public ValidationResult validate(RuntimeContainer container) {
         // also check that the properties is the right type
         if (validate.getStatus() != Result.ERROR) {
             if (!(properties instanceof TSnowflakeOutputProperties)) {
-                return new ValidationResult().setStatus(Result.ERROR)
-                        .setMessage("properties should be of type :" + TSnowflakeOutputProperties.class.getCanonicalName());
+                return new ValidationResult().setStatus(Result.ERROR).setMessage(
+                        i18nMessages.getMessage("error.wrongPropetiesType", TSnowflakeOutputProperties.class.getCanonicalName()));
             } // else this is the right type
```
Comment (francisu) [inline components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSink.java]: s/b wrongPropertiesType

Comment (francisu) [inline components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSink.java]: Also, this message as is is probably not helpful to a user who might get it. Think about how this situation might happen and show a message that gives the user more information.

File: components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSource.java [hunk 3]
```diff
@@ -55,7 +59,7 @@ public BoundedReader createReader(RuntimeContainer container) {
                 TalendRuntimeException.unexpectedException(e);
             }
         }
-        TalendRuntimeException.unexpectedException("Unknown properties: " + properties);
+        TalendRuntimeException.unexpectedException(i18nMessages.getMessage("error.unkownProperties", properties));
         return null;
```
Comment (francisu) [inline components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSource.java]: Is this an error a user would ever see? If not, then it should not be translated. If it is, then it should be something more clear and helpful.

File: components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeSink.properties [hunk 4]
```diff
@@ -0,0 +1 @@
+error.wrongPropetiesType=Properties should be of type : {0}
```
Comment (francisu) [inline components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeSink.properties]: No space before the ":"

File: components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeSource.properties [hunk 5]
```diff
@@ -0,0 +1 @@
+error.unkownProperties=Unknown properties:  {0}
```
Comment (francisu) [inline components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeSource.properties]: Only a single space after the ":"

**Comment by dmytro-sylaiev** [comment]
I18N tests are needed.

File: components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeConstants.properties [hunk 6]
```diff
@@ -0,0 +1,2 @@
+messages.success=Connection Successful
+error.incorrectAccount= Incorrect Snowflake Account was specified..
```
Comment (francisu) [inline components/components-snowflake/src/main/resources/org/talend/components/snowflake/runtime/SnowflakeConstants.properties]: Remove the ".." at the end

File: components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSource.java (hunk 3 above)
Comment (gonchar-ivan) [inline components/components-snowflake/src/main/java/org/talend/components/snowflake/runtime/SnowflakeSource.java]: I also think it is not a case that user can fix. Here, exception could be thrown only if Platform doesn't use component correctly. Thus, this message doesn't require i18n.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/189/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/189/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/191/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/191/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/195/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/195/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/197/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/197/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/202/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/202/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/206/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/206/)

**Pull request merged**